staticmap/staticmap.py

```diff
@@ -386,14 +389,14 @@ def _draw_base_layer(self, image):
                         # reached max tries to request tile
                         raise RuntimeError("could not download tile: {}: {}".format(self.url_template.format(z=self.zoom, x=tile_x, y=tile_y), res.status_code))
 
-                tile = Image.open(BytesIO(res.content))
+                tile = Image.open(BytesIO(res.content)).convert("RGBA")
                 box = [
                     self._x_to_px(x),
                     self._y_to_px(y),
                     self._x_to_px(x + 1),
                     self._y_to_px(y + 1),
                 ]
-                image.paste(tile, box)
+                image.paste(tile, box, tile)
```

Comment:
@christophlingg yeah, Im using staticmap to render previews for TMS sources, and some of the sources are just overlays, things like contour lines, and they require rendering on top of a white background to look good.